packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -223,46 +282,42 @@ const VotePage = ({
         availableVotes={availableVotes}
         vote={vote}
       />
-      <Col lg={{ span: 8, offset: 2 }}>
-        <Link to="/vote">← All Proposals</Link>
-      </Col>
-      <Col lg={{ span: 8, offset: 2 }} className={classes.proposal}>
+      <Col lg={{ span: 10, offset: 1 }} className={classes.proposal}>
         <div className="d-flex justify-content-between align-items-center">
-          <h3 className={classes.proposalId}>Proposal {proposal?.id}</h3>
-          <ProposalStatus status={proposal?.status}></ProposalStatus>
-        </div>
-        <div>
-          {startDate && startDate.isBefore(now) ? null : proposal ? (
-            <span>
-              Voting starts approximately {startDate?.format('MMMM D, YYYY h:mm A z')}{' '}
-              {startDate && `(${(startDate as any).fromNow()})`}{' '}
-            </span>
-          ) : (
-            ''
-          )}
-        </div>
-        <div>
-          {endDate && endDate.isBefore(now) ? (
-            <>
-              <div>Voting ended {endDate.format('MMMM D, YYYY h:mm A z')}</div>
-              <div>
-                This proposal has {quorumReached ? 'reached' : 'failed to reach'} quorum{' '}
-                {proposal?.quorumVotes !== undefined && `(${proposal.quorumVotes} votes)`}
-              </div>
-            </>
-          ) : proposal ? (
-            <>
-              <div>
-                Voting ends approximately {endDate?.format('MMMM D, YYYY h:mm A z')}{' '}
-                {endDate && `(${(endDate as any).fromNow()})`}{' '}
-              </div>
-              {proposal?.quorumVotes !== undefined && (
-                <div>A total of {proposal.quorumVotes} votes are required to reach quorum</div>
-              )}
-            </>
-          ) : (
-            ''
-          )}
+          <div className="d-flex justify-content-start align-items-start">
```

Comment:
What do you think about abstracting away this section into one `ProposalHeader` type component? Feels hard to read atm.